pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -0,0 +1,552 @@
+			let project4 = default_project_metadata(inst.get_new_nonce(), ISSUER_4);
+			let evaluations = default_evaluations();
+
+			inst.create_auctioning_project(project1, ISSUER_1, evaluations.clone());
+			inst.create_auctioning_project(project2, ISSUER_2, evaluations.clone());
+			inst.create_auctioning_project(project3, ISSUER_3, evaluations.clone());
+			inst.create_auctioning_project(project4, ISSUER_4, evaluations);
+		}
+
+		#[test]
+		fn plmc_price_change_doesnt_affect_evaluation_end() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let project_metadata = default_project_metadata(inst.get_new_nonce(), ISSUER_1);
+
+			// Decreasing the price before the end doesn't make a project over the threshold fail.
+			let target_funding =
+				project_metadata.minimum_price.saturating_mul_int(project_metadata.total_allocation_size);
+			let target_evaluation_usd = Percent::from_percent(10) * target_funding;
+
+			let evaluations = vec![(EVALUATOR_1, target_evaluation_usd).into()];
+			let evaluation_plmc = MockInstantiator::calculate_evaluation_plmc_spent(evaluations.clone());
+			let evaluation_existential = evaluation_plmc.accounts().existential_deposits();
+			inst.mint_plmc_to(evaluation_plmc);
+			inst.mint_plmc_to(evaluation_existential);
+
+			let project_id = inst.create_evaluating_project(project_metadata.clone(), ISSUER_1);
+			inst.evaluate_for_users(project_id, evaluations.clone()).unwrap();
+
+			let old_price = <TestRuntime as Config>::PriceProvider::get_price(PLMC_FOREIGN_ID).unwrap();
+			PRICE_MAP.with_borrow_mut(|map| map.insert(PLMC_FOREIGN_ID, old_price / 2.into()));
+
+			inst.start_auction(project_id, ISSUER_1).unwrap();
+
+			// Increasing the price before the end doesn't make a project under the threshold succeed.
+			let evaluations = vec![(EVALUATOR_1, target_evaluation_usd / 2).into()];
+			let evaluation_plmc = MockInstantiator::calculate_evaluation_plmc_spent(evaluations.clone());
+			let evaluation_existential = evaluation_plmc.accounts().existential_deposits();
+			inst.mint_plmc_to(evaluation_plmc);
+			inst.mint_plmc_to(evaluation_existential);
+
+			let project_id = inst.create_evaluating_project(project_metadata.clone(), ISSUER_2);
+			inst.evaluate_for_users(project_id, evaluations.clone()).unwrap();
+
+			let old_price = <TestRuntime as Config>::PriceProvider::get_price(PLMC_FOREIGN_ID).unwrap();
+			PRICE_MAP.with_borrow_mut(|map| map.insert(PLMC_FOREIGN_ID, old_price * 2.into()));
+
+			let update_block = inst.get_update_block(project_id, &UpdateType::EvaluationEnd).unwrap();
+			let now = inst.current_block();
+			inst.advance_time(update_block - now + 1).unwrap();
+			let project_status = inst.get_project_details(project_id).status;
+			assert_eq!(project_status, ProjectStatus::EvaluationFailed);
+		}
+	}
+
+	#[cfg(test)]
+	mod failure {
+		use super::*;
+
+		#[test]
+		fn round_fails_after_not_enough_bonds() {
```

Comment:
Not sure if this function is necessary? We are already testing this case in a way in the test above. Or do you think its important for seperation of concerns.

Reply:
This is a very critical component we need to test so I don't mind implicitly testing it twice if the trade-off is better readability